relay.go

```diff
@@ -60,7 +62,8 @@ type EnvConfig struct {
 	ApiKey             string // deprecated, equivalent to SdkKey
 	MobileKey          *string
 	EnvId              *string
-	Prefix             string
+	Prefix             string // used only if Redis, Consul, or DynamoDB is enabled
```

Comment:
Might be better to use pointer types so that you can check for `nil` instead of empty string (`""` seems like it could be a valid prefix).

Reply:
Actually nevermind. This is fine. I misread some of the code below initially 🤦‍♂️